### PR TITLE
Add some clarifications to the "Convert HTML to AMP" Tutorial

### PR DESCRIPTION
**Pull request by mrjoro**
In preparation for the Roadshow I walked through the "Convert HTML to AMP" tutorial and made some changes/additions that may make things more clear for people not familiar with AMP.

cc/ @pbakaus

### CODE REVIEW

File: content/docs/tutorials/converting/discoverable.md [hunk 1]
```diff
@@ -81,7 +85,7 @@ For our news article, **add** the following structured data to the bottom of the
 
 
 {% call callout('Note', type='note') %}
-The content should always be the same. For news articles, specify the “NewsArticle” type. The headline should match your article’s title. The image object refers to the hero image of the article.
+The context should always be the same. For news articles, specify the “NewsArticle” type. The headline should match your article’s title. The image object refers to the hero image of the article.
```
Comment (bpaduch) [inline content/docs/tutorials/converting/discoverable.md]: I think "content" should remain.  It's the content of the structured data that remains the same, but the type needs to change.

Comment (pbakaus) [reply content/docs/tutorials/converting/discoverable.md]: +1.

File: content/docs/tutorials/converting/resolving-errors.md [hunk 2]
```diff
@@ -124,7 +126,7 @@ The problem is that this is an external stylesheet reference. In AMP, to keep th
 
 So, let's resolve the error:
 
-1.  **Remove** the `<link>` tag in the `<head>` and replace it with an inline `<style amp-custom></style>` tag. The `amp-custom` attribute on the style tag is mandatory.
+1.  **Remove** the stylesheet `<link>` tag in the `<head>` and replace it with an inline `<style amp-custom></style>` tag. The `amp-custom` attribute on the style tag is mandatory.
```
Comment (pbakaus) [inline content/docs/tutorials/converting/resolving-errors.md]: "remove the `<link>` tag pointing to the stylesheet"

File: content/docs/tutorials/converting/building-page.md [hunk 3]
```diff
@@ -42,7 +42,9 @@ Your `article.amp.html` file should look like the following:
 
 This is an intentionally simplistic page with common static news article elements: CSS, JavaScript, and an image tag.
 
-Our AMP version of the article is just a copy of the original article right now. Let's convert it to an AMP. To begin, we will add the AMP JavaScript library file and view what errors appear when the AMP validator is turned on.
+Our AMP version of the article is just a copy of the original article right now. Let's convert it to an AMP.
+
+To begin, we will add the AMP JavaScript library file.  This alone won't make your new file a valid AMP page, but we'll see below how the AMP JavaScript can help us figure out what we need to do to fix that.
```
[the remark this answers is not in the document]
Comment (mrjoro) [reply content/docs/tutorials/converting/building-page.md]: Done.

File: content/docs/tutorials/converting/building-page.md [hunk 4]
```diff
@@ -70,14 +72,13 @@ For example:
...
-
-In the Developer Console, you should receive several validation errors:
+In the Developer Console, you should receive several validation errors (you may need to manually refresh the page in your browser to see these):
 
 {{ image('/static/img/docs/tutorials/tut-convert-html-validation-errors.png', 905, 427, align='', caption='AMP validation errors for our sample') }}
 
+In order to make this a valid AMP document we're going to have to fix all of these errors--which is exactly what we'll be doing in this codelab.
 
-As we are working with a mobile news article, let's **simulate** a mobile device experience in the browser's developer tools. For example, in Chrome Developer Tools, click the mobile phone icon, and select a mobile device from the menu.
+Before we do that, let's **simulate** a mobile device experience in the browser's developer tools since we are working with a mobile news article.  For example, in Chrome Developer Tools, click the mobile phone icon, and select a mobile device from the menu.
```
[the remark this answers is not in the document]
Comment (mrjoro) [reply content/docs/tutorials/converting/building-page.md]: Done.

File: content/docs/tutorials/converting/discoverable.md [hunk 5]
```diff
@@ -7,13 +7,17 @@ Now that you've made a news article in AMP, let's ensure users can find and disc
...
+Canonical linking in regular HTML pages is a common technique for declaring which page should be considered the preferred page when multiple pages include the same content.
+
+One common approach when adding AMP to a website is to generate AMP versions of traditional non-AMP HTML pages.  Both versions have generally the same content (e.g. the text of an article) but they may have different presentations.  In this scenario you should treat the traditional HTML pages as the “canonical” pages and pair the AMP pages with those HTML pages.
+
+It's possible (and often preferable) to create websites entirely out of AMP documents with no non-AMP version of the documenst.  In this case an AMP page is considered to be the canonical version.
```
[the remark this answers is not in the document]
Comment (mrjoro) [reply content/docs/tutorials/converting/discoverable.md]: Done.

File: content/docs/tutorials/converting/discoverable.md (hunk 1 above)
Comment (mrjoro) [reply content/docs/tutorials/converting/discoverable.md]: Got it, though I'm not sure I understand what we're trying to say by "the content should always be the same."  Don't we want developers to change the other fields as well (like the datePublished, the author, etc.)?

File: content/docs/tutorials/converting/resolving-errors.md (hunk 2 above)
Comment (mrjoro) [reply content/docs/tutorials/converting/resolving-errors.md]: Done.

**Pull request merged**